client/src/app/site/pages/meetings/pages/projectors/projectors.subscription.ts

```diff
@@ -61,14 +66,16 @@ export const getProjectorListSubscriptionConfig: SubscriptionConfigGenerator = (
                     }
                 ]
             },
-            `projector_countdown_ids`,
-            `projector_message_ids`,
-            ...MEETING_DEFAULT_PROJECTOR_IDS_KEYS,
-            { idField: `speaker_ids`, additionalFields: [`meeting_user_id`] },
-            `list_of_speakers_ids`,
-            { idField: `agenda_item_ids`, fieldset: [`item_number`, `content_object_id`] }
-        ],
-        additionalFields: [`reference_projector_id`]
```

Comment:
This seems to be missing.

Reply:
I moved it to line 28

Reply:
`additionalFields` was actually unnecessary. You don't need to subscribe additional fields when already subscribing ALL fields.